packages/trail-hapi-server/package.json

```diff
@@ -38,21 +38,21 @@
     "coverage": "lab -l -c test",
     "depcheck": "npx depcheck --ignores=config",
     "start": "node start.js",
-    "test": "lab test --globals __extends,__assign,__rest,__decorate,__param,__metadata,__awaiter,__generator,__exportStar,__values,__read,__spread,__spreadArrays,__await,__asyncGenerator,__asyncDelegator,__asyncValues,__makeTemplateObject,__importStar,__importDefault,__classPrivateFieldGet,__classPrivateFieldSet"
+    "test": "lab test --globals __core-js_shared__,CSS,regeneratorRuntime,__extends,__assign,__rest,__decorate,__param,__metadata,__awaiter,__generator,__exportStar,__values,__read,__spread,__spreadArrays,__await,__asyncGenerator,__asyncDelegator,__asyncValues,__makeTemplateObject,__importStar,__importDefault,__classPrivateFieldGet,__classPrivateFieldSet"
   },
   "dependencies": {
-    "@hapi/hapi": "^19.1.0",
-    "@hapi/inert": "^6.0.1",
+    "@hapi/hapi": "^19.2.0",
```

Comment:
Can you also bump Hapi to latest? So we can ship a new major.

Reply:
Done in 1ba66a91dd9d3fa8d96b9215c27a4f4243f019ef